crypto/types/src/types.rs

```diff
@@ -100,17 +100,77 @@ mod arrays {
 }
 
 /// Represent an uncompressed elliptic curve point on the secp256k1 curve
+///
+/// ```rust
+/// # use hopr_crypto_types::prelude::*;
+/// # use hex_literal::hex;
+/// # use k256::{elliptic_curve::NonZeroScalar, Scalar, Secp256k1};
```

Comment:
```suggestion
/// use hopr_crypto_types::prelude::*;
/// use hex_literal::hex;
/// use k256::{elliptic_curve::NonZeroScalar, Scalar, Secp256k1};
```

Reply:
In docstrings, it is necessary to make the code compile but these lines are excluded from rendering for better observability